tests/fixtures/jats/02791-v1-vor.xml

```diff
@@ -0,0 +1 @@
ifferent and make decisions about promotions accordingly? How many of us would not ask questions of investigators in their own fields about how the data were collected, analysed and verified? And yet how many of us have the foggiest idea how exactly the numbers of citations for publications have been collected, and what verification procedures are used?</p><disp-quote><p>How many of us have the foggiest idea how exactly the numbers of citations for publications have been collected?</p></disp-quote><p>One might argue that the actual numbers do not matter, and it is only orders of magnitude at issue. If so, why even bother with the numbers? At the limit, the numbers just validate what we already know. Someone like Linda Buck or Eric Kandel has more citations than any assistant professor. But we don’t need indexes to tell us that Nobel Prize winners have had impact and influence on their fields, and that scientists who have only been in the field for 10 years have had less cumulative effect.</p><p>When I was three years old I got my head stuck between the bars of a playground in Central Park because I was curious if my head was larger or smaller than the distance between the bars. As I stood there unable to get my head out of the bars, I felt very stupid, as I realised I could have measured the distance between the bars by holding my hands apart, without getting stuck. It was my first real experience of feeling truly self-consciously dumb. Many years later I feel equally stupid for believing that the numbers that have become so important to so many careers are reliable. I have never believed that these numbers are wisely used in assessments, but now I know there is all the more reason to forget all of these indexes and metrics and to focus on the science.</p></body><back><fn-group content-type="competing-interest"><fn fn-type="conflict" id="conf1"><label>Competing interests:</label><p>The author declares that no competing interests exist.</p></fn></fn-group><ref-list><title>References</title><ref id="bib1"><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hooper</surname><given-names>SL</given-names></name><name><surname>Marder</surname><given-names>E</given-names></name></person-group><year>1987</year><article-title>Modulation of the lobster pyloric rhythm by the peptide proctolin</article-title><source>Journal of Neuroscience</source><volume>7</volume><fpage>2097</fpage><lpage>2112</lpage></element-citation></ref><ref id="bib2"><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Marder</surname><given-names>E</given-names></name><name><surname>Calabrese</surname><given-names>RL</given-names></name></person-group><year>1996</year><article-title>Principles of rhythmic motor pattern generation</article-title><source>Physiological Reviews</source><volume>76</volume><fpage>687</fpage><lpage>717</lpage></element-citation></ref></ref-list></back></article>
```

Comment:
We don't need to add a new article for this as we have a few instances in the XML we already have. In fact, there should be in ```tests/fixtures/html/02619-v1-vor-section-main-text.html```